inc/update.class.php

```diff
@@ -399,13 +399,15 @@ public function doUpdates($current_version = null) {
          case "9.2":
             include_once("{$updir}update_92_921.php");
             update92to921();
-            break;
 
          case "9.2.1":
-         case GLPI_VERSION:
-         case GLPI_SCHEMA_VERSION:
+         case "9.2.2":
```

Comment:
Maybe I am wrong, but update to `9.2.2` should be done only if current version is lower than or equal to `9.2.1`.

Reply:
Indeed, it should... But since we've missed the update; an upgrade from 9.2.2 must run this one.

Reply:
My bad.